static/app/utils/replays/createUrlToShare.tsx

```diff
@@ -0,0 +1,6 @@
+function createUrlToShare(currentTime: number) {
```

Comment:
Let's change this parameter name to include the time units (ms it looks like?)

```suggestion
function createUrlToShare(currentTimeMs: number) {
```